custom/index.html

```diff
@@ -625,8 +625,6 @@
         <script type="text/javascript" src="/js/damage_calc.js"></script>
         <script type="text/javascript" src="/js/load.js"></script>
         <script type="text/javascript" src="/js/load_ing.js"></script>
-        <script type="text/javascript" src="/js/crafter.js"></script>      
-        <script type="text/javascript" src="/js/craft.js"></script>    
```

Comment:
Must fix: These are required for customizer to be able to take crafted item as base.

Reply:
Resolved with [3aa1a56](https://github.com/hppeng-wynn/hppeng-wynn.github.io/pull/284/commits/3aa1a56aee5729beee0f28146c6e08237485895b)